src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,6 @@
+spring.datasource.url=jdbc:h2:mem:${database}
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=${user}
+spring.datasource.password=${password}
```

Comment:
in case of `prod` profile it makes sense to hide DB connection credentials. But I don't think it does on dev profile. They could be stored here in order to make things easier.